.vscode/settings.json

```diff
@@ -1,3 +1,10 @@
 {
-    "typescript.tsdk": "node_modules/typescript/lib"
-}
\ No newline at end of file
+    "typescript.tsdk": "node_modules/typescript/lib",
+    "eslint.validate": [
+        "javascript",
+        "javascriptreact",
+        "typescript",
+        "typescriptreact"
+    ],
```

Comment:
For en uinnvidd så det litt rart ut at dette ble gjentatt to ganger. Copy - paste - paste?

Reply:
Det er 2 for js og 2 for ts.